### PR TITLE
Tweak dashboard grid to give projects/tickets equal height.

### PR DESCRIPTION
**Pull request by jarosenb**
## Overview:
Update the Projects/tickets column in the dashboard to use its own grid with equal heights. This fixes an issue where the heights were getting calculated in a weird way due to the fact that the sysmon needs to set its own height.

## UI:
image

### CODE REVIEW

File: apps/tup-ui/src/pages/Dashboard/Dashboard.module.css [hunk 1]
```diff
@@ -32,8 +32,31 @@
...
+
+
+.project-ticket-grid {
+  display: grid;
+  gap: 25px;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr;
+  grid-template-areas: 
+    "_projects"
+    "_tickets";
```
Comment (wesleyboar) [inline apps/tup-ui/src/pages/Dashboard/Dashboard.module.css:49]: Why do you prefix these area names with an underscore? Was there conflict? (I would not expect conflict).

Comment (wesleyboar) [reply apps/tup-ui/src/pages/Dashboard/Dashboard.module.css:49]: Oh, I think I see. The area name `projects` (parent) is confusing. Hm… if only it were named more inclusively… I don't know how to name it without getting verbose...

<details><summary>idea: "projects_tickets"</summary>

```
    grid-template-areas:
      "projects_tickets systems"
      "projects_tickets updates"
      "projects_tickets updates"
      "projects_tickets updates";
```
```
  grid-template-areas: 
    "projects"
    "tickets";
```

</details>
<details><summary>idea: "other"</summary>

```
    grid-template-areas:
      "other systems"
      "other updates"
      "other updates"
      "other updates";
```
```
  grid-template-areas: 
    "projects"
    "tickets";
```
</details>
<details><summary>idea: 2 columns (1 parent grid or flex; 2 child grids)</summary>

```
          <div className={styles['news-system-grid']}>
            <UserNews />
            <SystemMonitor />
          </div>
          <div className={styles['project-ticket-grid']}>
            <ProjectsDashboard />
            <TicketsDashboard />
          </div>
```
</details>

**Review by wesleyboar — APPROVED** [review]
This works. I think it could be made more intuitive, but I don't have a surefire solution. I've made three suggestions in one comment.

File: apps/tup-ui/src/pages/Dashboard/Dashboard.module.css (hunk 1 above)
Comment (jarosenb) [reply apps/tup-ui/src/pages/Dashboard/Dashboard.module.css]: Thanks, I went with the projects_tickets idea.

**Pull request merged**